app/components/portfolios/details_component.sass

```
@@ -0,0 +1,39 @@
+$status_not-set: var(--progressBar-track-bgColor) // invisible background color
+
+@mixin portfolio-sub-item-status-background-color
+  &.-not-set
+    background-color: $status_not-set
+  &.-off-track
+    background-color: $project-status_off-track
+  &.-at-risk
+    background-color: $project-status_at-risk
+  &.-on-track
+    background-color: $project-status_on_track
```

TIL that sass doesn't differentiate underscores and dashes, but for findability:
```suggestion
    background-color: $project-status_on-track
```